diagnostic.md

```diff
@@ -62,7 +62,7 @@ var result = strangeAdd(0, -5);
 
 What is the value of result?  Explain your answer.
 
-_Replace this text with your answer._
+5. The second return is evaluated b/c the first condition is not met.
```

Comment:
Correct, the second return is evaluated but you left out the value of the result. The value is in fact 5.